src/server/utils/WireGuard.ts

```diff
@@ -417,90 +300,113 @@ Endpoint = ${WG_HOST}:${WG_CONFIG_PORT}`;
     clientId: string;
     expireDate: string | null;
   }) {
-    const client = await this.getClient({ clientId });
+    let updatedDate: Date | null = null;
 
     if (expireDate) {
       const date = new Date(expireDate);
       date.setHours(23);
       date.setMinutes(59);
       date.setSeconds(59);
-      client.expireAt = date.toISOString();
-    } else {
-      client.expireAt = null;
+      updatedDate = date;
     }
-    client.updatedAt = new Date().toISOString();
 
-    await this.saveConfig();
-  }
+    await Database.updateClientExpirationDate(clientId, updatedDate);
 
-  async __reloadConfig() {
-    await this.__buildConfig();
-    await this.__syncConfig();
+    await this.saveConfig();
   }
 
-  async restoreConfiguration(config: string) {
-    debug('Starting configuration restore process.');
+  // TODO: reimplement database restore
+  async restoreConfiguration(_config: string) {
+    /* DEBUG('Starting configuration restore process.');
     // TODO: sanitize config
     const _config = JSON.parse(config);
     await this.__saveConfig(_config);
     await this.__reloadConfig();
-    debug('Configuration restore process completed.');
+    DEBUG('Configuration restore process completed.'); */
   }
 
+  // TODO: reimplement database restore
   async backupConfiguration() {
-    debug('Starting configuration backup.');
+    /* DEBUG('Starting configuration backup.');
     const config = await this.getConfig();
     const backup = JSON.stringify(config, null, 2);
-    debug('Configuration backup completed.');
-    return backup;
+    DEBUG('Configuration backup completed.');
+    return backup; */
+  }
+
+  async Startup() {
+    // TODO: improve this
+    await new Promise((res) => {
+      function wait() {
+        if (Database.connected) {
+          return res(true);
+        }
+      }
+      setTimeout(wait, 1000);
+    });
```

Comment:
@tetuaoro  
Database migration takes some time. Either we make it manual or leave in the "hacky" wait for connection. Even tho on every start there are some errors as the database takes a while to startup and `#db` is undefined